http/http_connection.hpp

```diff
@@ -579,21 +576,6 @@ class Connection :
 
                 boost::beast::http::verb method = parser->get().method();
                 readClientIp();
-                boost::urls::error_code uriEc;
-                boost::urls::string_view uriStringView(
-                    parser->get().target().data(),
-                    parser->get().target().size());
-                BMCWEB_LOG_DEBUG << "Parsing URI: " << uriStringView;
-                req->urlView =
-                    boost::urls::parse_relative_ref(uriStringView, uriEc);
-                if (uriEc)
-                {
-                    BMCWEB_LOG_ERROR << "Failed to parse URI "
-                                     << uriEc.message();
-                    return;
-                }
-                req->url = std::string_view(req->urlView.encoded_path().data(),
-                                            req->urlView.encoded_path().size());
```

Comment:
I think there is a bug here - we need to also remove the req->urlParams related code below at line 594 till 603 because we dont have the req->urlView populated after removing this code chunk

Reply:
I still see 
```
                    req->urlParams = req->urlView.query_params();
#ifdef BMCWEB_ENABLE_DEBUG
                    std::string paramList = "";
                    for (const auto param : req->urlParams)
                    {
                        paramList += param->key() + " " + param->value() + " ";
                    }
                    BMCWEB_LOG_DEBUG << "QueryParams: " << paramList;
#endif
```
upstream at https://github.com/openbmc/bmcweb/blob/master/http/http_connection.hpp#L588

We could remove this, should also remove: 
https://github.com/openbmc/bmcweb/blob/597d2b142362bafa90f24fc8c30750afab91f78f/http/http_request.hpp#L26

Reply:
> Tested: GUI works including requests like powering on

Ok. Since this worked fine during your tests

Reply:
I removed this, just an FYI